### ISSUE TITLE
Release a new version to Crates.io to use cpal with ndk 0.8

### ISSUE THREAD

**Issue by ameknite** [RustAudio/cpal#841]
# Objective:
Use cpal with ndk 0.8

In #820 ndk was updated to 0.8. but a new version to Crates.io has not being released.

alternatives:

```toml
[dependencies]
cpal = { git = "https://github.com/RustAudio/cpal", branch = "master" }
```

**Comment by evilpie** [comment]
Additionally updating `alsa` to 0.9 would get rid of the outdated `nix` dependency.

**Comment by ameknite** [comment]
fixed, 0.15.3 has been released
https://github.com/RustAudio/cpal/pull/856
